src/test/resources/arquillian.xml

```diff
@@ -1,13 +1,9 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns="http://jboss.org/schema/arquillian"
-            xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
 
-    <defaultProtocol type="Servlet 3.0" />
-
-    <container qualifier="wildfly-managed" default="true">
+<arquillian>
+    <container qualifier="lra-test-container" default="true" mode="suite">
         <configuration>
-            <property name="jbossHome">${jboss.home}</property>
+            <property name="javaVmArguments">-Dthorntail.http.port=8180 -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=9000 <!-- -Dthorntail.logging=TRACE--></property>
```

Comment:
why does we need to run arqullian on port 8180? and please remove debug configuration